### PR TITLE
Added TCP Port setting for SSH. 

### PR DESCRIPTION
**Pull request by richardschoen**

### CODE REVIEW

**Comment by richardschoen** [comment]
Also change temp file name from /tmp/randomvalue to /tmp/vscodetemp-randomvalue so they can be cleaned up easily. Perhaps home directory makes sense at some point but at least they can clean their temp dirs easier with this naming.

File: src/api/IBMi.js [hunk 1]
```diff
@@ -17,7 +18,7 @@ module.exports = class IBMi {
   }
 
   /**
-   * @param {{host: string, username: string, password: string, keepaliveInterval?: number}} connectionObject 
+   * @param {{host: string, port: string, username: string, password: string, keepaliveInterval?: number}} connectionObject
```
Comment (worksofliam) [inline src/api/IBMi.js:21]: Why is port a string? Shouldn't it b a number?

**Review by worksofliam — APPROVED** [review]
LGTM. Need to do some testing but then I'll give it a whirl.

File: package.json [hunk 2]
```diff
@@ -25,84 +27,100 @@
...
-									}
+						"type": "object",
+						"title": "Connection details",
+						"properties": {
+							"host": {
+								"type": "string",
+								"description": "Host name or IP address"
+							},
+							"port": {
+								"type": "string",
```
Comment (worksofliam) [inline package.json:71]: This should be a number.

**Review by worksofliam — CHANGES_REQUESTED** [review]
After the user is done in the webview, the `port` needs to be converted to a number.

Please make those changes and I can review again.

File: src/api/IBMi.js (hunk 1 above)
Comment (richardschoen) [reply src/api/IBMi.js:21]: I guess, but it works. Must get auto-converted.

Comment (worksofliam) [reply src/api/IBMi.js:21]: It may work, but I am trying my best to write the easier code to understand + eslint is moaning at me! 
image

**Comment by richardschoen** [comment]
Updates pushed

**Comment by worksofliam** [comment]
Accidently put it onto another branch.

**Pull request closed without merging**